test/unit/gateways/securion_pay_test.rb

```diff
@@ -0,0 +1,858 @@
+      <- "amount=2000&currency=usd&card[number]=4242424242424242&card[expMonth]=9&card[expYear]=2016&card[cvc]=123&card[cardholderName]=Longbob+Longsen&description=ActiveMerchant+test+charge&metadata[email]=foo%40example.com"
+      -> "HTTP/1.1 200 OK\r\n"
+      -> "Server: cloudflare-nginx\r\n"
+      -> "Date: Thu, 19 Mar 2015 23:16:59 GMT\r\n"
+      -> "Content-Type: application/json;charset=UTF-8\r\n"
+      -> "Transfer-Encoding: chunked\r\n"
+      -> "Connection: close\r\n"
+      -> "Set-Cookie: __cfduid=d2825af5ba35f4e682da0c51abb33a5d91426807018; expires=Fri, 18-Mar-16 23:16:58 GMT; path=/; domain=.securionpay.com; HttpOnly\r\n"
+      -> "CF-RAY: 1c9ce35b76a805d5-WAW\r\n"
+      -> "Content-Encoding: gzip\r\n"
+      -> "\r\n"
+      -> "154\r\n"
+      reading 340 bytes...
+      -> "[GZIP_ENCODED_CONTENT_REMOVED_INTENTIONALLY]"
+      read 340 bytes
+      reading 2 bytes...
+      -> "\r\n"
+      read 2 bytes
+      -> "a\r\n"
+      reading 10 bytes...
+      -> "[GZIP_ENCODED_CONTENT_REMOVED_INTENTIONALLY]"
+      read 10 bytes
+      reading 2 bytes...
+      -> "\r\n"
+      read 2 bytes
+      -> "0\r\n"
+      -> "\r\n"
+      Conn close
+    PRE_SCRUBBED
+  end
+
+  # When updating post_scrubbed response below, please remove gzip-encoded contents
+  # as they contain non-UTF-8 characters that will raise exception when scrubbing.
+  def post_scrubbed
+    <<-POST_SCRUBBED
+      opening connection to api.securionpay.com:443...
+      opened
+      starting SSL for api.securionpay.com:443...
+      SSL established
+      <- "POST /charges HTTP/1.1\r\nContent-Type: application/x-www-form-urlencoded\r\nAuthorization: Basic [FILTERED]\r\nUser-Agent: SecurionPay/v1 ActiveMerchantBindings/1.47.0\r\nAccept-Encoding: gzip;q=1.0,deflate;q=0.6,identity;q=0.3\r\nAccept: */*\r\nConnection: close\r\nHost: api.securionpay.com\r\nContent-Length: 214\r\n\r\n"
+      <- "amount=2000&currency=usd&card[number]=[FILTERED]&card[expMonth]=9&card[expYear]=2016&card[cvc]=[FILTERED]&card[cardholderName]=Longbob+Longsen&description=ActiveMerchant+test+charge&metadata[email]=foo%40example.com"
+      -> "HTTP/1.1 200 OK\r\n"
+      -> "Server: cloudflare-nginx\r\n"
+      -> "Date: Thu, 19 Mar 2015 23:16:59 GMT\r\n"
+      -> "Content-Type: application/json;charset=UTF-8\r\n"
+      -> "Transfer-Encoding: chunked\r\n"
+      -> "Connection: close\r\n"
+      -> "Set-Cookie: __cfduid=d2825af5ba35f4e682da0c51abb33a5d91426807018; expires=Fri, 18-Mar-16 23:16:58 GMT; path=/; domain=.securionpay.com; HttpOnly\r\n"
+      -> "CF-RAY: 1c9ce35b76a805d5-WAW\r\n"
+      -> "Content-Encoding: gzip\r\n"
+      -> "\r\n"
+      -> "154\r\n"
+      reading 340 bytes...
+      -> "[GZIP_ENCODED_CONTENT_REMOVED_INTENTIONALLY]"
```

Comment:
Yes, it is. Should it be a default header sent with all requests to the API? At the moment I just recorded the transcript with gzip disabled and replaced it here, but did not add the header to the requests.